server/events/plan_command_runner.go

```diff
@@ -189,10 +189,11 @@ func (p *PlanCommandRunner) run(ctx *command.Context, cmd *CommentCommand) {
 
 	if len(projectCmds) == 0 && p.SilenceNoProjects {
 		ctx.Log.Info("determined there was no project to run plan in")
-		if !p.silenceVCSStatusNoProjects {
+		if !p.silenceVCSStatusNoProjects && !cmd.IsForSpecificProject() {
```

Comment:
Can this block be tested in `plan_command_runner_test.go` ?

Reply:
I added a test going through most of the paths